src/streamlink_cli/main.py

```diff
@@ -977,11 +1006,15 @@ def setup_logging(stream=sys.stdout, level="info"):
 
 
 def main():
+    global use_win32exit
```

Comment:
This isn't required is it? If you're just reading a `global` you don't need to declare it `global`, right?

Reply:
You are right. Changed.